src/System.Memory/src/System/Buffers/Text/Utf8Parser/Utf8Parser.Integer.Unsigned.cs

```diff
@@ -77,7 +77,7 @@ public static bool TryParse(ReadOnlySpan<byte> text, out ushort value, out int b
         {
             switch (standardFormat)
             {
-                case (default):
+                case default(char):
```

Comment:
@VSadov, why was this change necessary as part of the compiler update? `case (default)` compiled previously.

Reply:
https://github.com/dotnet/roslyn/pull/23629